src/Symfony/Component/Security/Guard/AuthenticatorInterface.php

```diff
@@ -90,7 +90,7 @@ public function getUser($credentials, UserProviderInterface $userProvider);
      *
      * @param mixed $credentials
      *
-     * @return bool
+     * @return mixed true if the credentials are valid, anything else otherwise.
```

Comment:
Now that #33308 has been merged, we should revert this change.

Reply:
reverted